libraries/c_sdk/standard/mqtt/CODESIZE.md

```diff
@@ -0,0 +1,45 @@
+# Code size of MQTT Compatibility Layer
+
+Code size for the MQTT Compatibility Layer is calculated using GCC for ARM for cortex-M4 on [202011.00 release](https://github.com/aws/amazon-freertos/releases/tag/202011.00). Build for calculating code size is done using build configuration for [cypress CY8CKIT-064S0S2-4343W](https://docs.aws.amazon.com/freertos/latest/userguide/getting_started_cypress_psoc64.html) by using the `Release` configuration. All the logs were disabled in this build. Refer to the table below for calculated code sizes.
```

Comment:
Code size for the MQTT Compatibility Layer is calculated using GCC for ARM Cortex-M4 on [FreeRTOS 202011.00 release]. Build for calculating code size is done using build configuration for [Cypress CY8CKIT-064S0S2-4343W] by using the `Release` configuration. All logs were disabled in this build. Refer to the table below for calculated code sizes.

Reply:
Updated the comments. Thanks